tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -37,9 +37,9 @@
 # response. To achieve this we pick all the channels that are not considered
 # to be short (less than 1 cm distance between optodes).
 
-short_channels = mne.preprocessing._beer_lambert_law._short_channels(
+is_short = mne.preprocessing.short_channels(
     raw_intensity, threshold=0.01)
-long_channels = np.logical_not(short_channels)
+long_channels = np.logical_not(is_short)
```

Comment:
FYI this can just be `~is_short`, assuming `is_short` is `bool` dtype

Reply:
That's what I had but I changed it based on [this suggestion](https://github.com/mne-tools/mne-python/pull/7018#discussion_r342721129). But I am fine with it either way, whatever you all prefer.

Reply:
given what you do here I would even expose `_probe_distances` under the name `nirs_source_detector_distances` 

```
picks = mne.pick_types(raw.info, meg=False, fnirs=True)
dists = nirs_source_detector_distances(raw.info, picks=picks)
raw_intensity.pick(picks[dist < 0.01])
```